### ISSUE TITLE
adaptive card image alt text option if not loaded? 

### ISSUE THREAD

**Issue by liangchr** [MicrosoftDocs/msteams-docs#1945]
If there is a bad image link (or a customer has come proxy for my image link) on an adaptive card, can I specify alt text? Right now the customer is just getting undefined bc of a proxy with https://urlp.sfbassets.com

I want to specify alt text on image load fail. 
so I don't get this: 
<img width="276" alt="Screen Shot 2020-08-04 at 3 14 10 PM" src="https://user-images.githubusercontent.com/10538763/89351156-42909000-d666-11ea-915d-726b913c09df.png">

**Comment by Trinetra-MSFT** [comment]
@liangchr , Could you please confirm how you are displaying the images in adaptive card? Adaptive card do support data URI in base 64 format try below piece of code
```
{
    "type": "AdaptiveCard",
    "body": [
        {
            "type": "Image",
            "style": "ImageTest",
            "url": "data:image/png;base64,iVBORw0KGgoAAA ANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4 //8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU 5ErkJggg==",
            "size": "Medium"
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.0"
}
```

**Comment by liangchr** [comment]
@Trinetra-MSFT Thanks for the support. I did not know it supported alt text. Is there documentation for this anywhere? Ive been referring to: https://adaptivecards.io/explorer/TextBlock.html but it had nothing about alt text on it. For some reason when I use the JSON you sent me I am getting an error from the bot service.

**Comment by liangchr** [comment]
HI @Trinetra-MSFT appreciate the help. Could you explain one thing to me? When I used this dataURL image type and inspect the image on the adaptive card in the dev tools it looks like the image is still being loaded through a URL: https://urlp.asm.skype.com/. We have a customer that has a proxy enabled preventing requests from urp.asm.skype. Is there any way we can load this image whilst side-stepping this URL? or at the very least be able to specify alt text (like <image alt="image description"><image>) for when the image cannot be loaded for the customer? Right now the adaptive card image alt txt value is "undefined"
<img width="417" alt="Screen Shot 2020-08-06 at 2 26 49 PM" src="https://user-images.githubusercontent.com/10538763/89584420-e8700600-d7f0-11ea-9334-8cd04fe8106b.png">

**Comment by Trinetra-MSFT** [comment]
@liangchr , Teams is blocking that image url due to security concerns, better to put the images in solution or fetch it from sharepoint library

**Comment by liangchr** [comment]
@Trinetra-MSFT Thanks for the clarification. So if I fetch the images from sharepoint the images won't load from "https://urlp.asm.skype.com/?" (the customer is blocking urlp.asm.skype but maybe not "sharepoint.com" so this _might_ be a solution?)

**Comment by liangchr** [comment]
@Trinetra-MSFT Also want to clarify: is specifying alt text like `<img alt="dfdfd">` possible?

**Comment by Trinetra-MSFT** [comment]
@liangchr , Please look at Adaptive Card image support: [https://adaptivecards.io/explorer/Image.html](https://adaptivecards.io/explorer/Image.html). you can use altText property for describing the alternative text for image
